### PR TITLE
showInspector is passing in an isGlobal flag - but plugin was not making...

### PR DESCRIPTION
**Pull request by wmathurin**
... use of it

### CODE REVIEW

File: libs/SalesforceHybridSDK/SalesforceHybridSDK/Classes/Plugins/SFSmartStore/SFSmartStorePlugin.h [hunk 1]
```diff
@@ -24,30 +24,14 @@
...
 #import <Foundation/Foundation.h>
 #import "SFForcePlugin.h"
+#import <SalesforceSDKCore/SFStoreCursor.h>
 
 /**
  String used with Cordova to uniquely identify this plugin
  */
 extern NSString * const kSmartStorePluginIdentifier;
 
-@class SFStoreCursor;
```
Comment (wmathurin) [inline libs/SalesforceHybridSDK/SalesforceHybridSDK/Classes/Plugins/SFSmartStore/SFSmartStorePlugin.h]: Cleaning up - we don't need those exposed as properties

File: libs/SalesforceHybridSDK/SalesforceHybridSDK/Classes/Plugins/SFSmartStore/SFSmartStorePlugin.m [hunk 2]
```diff
@@ -316,7 +310,9 @@ - (void)pgReIndexSoup:(CDVInvokedUrlCommand *)command
 - (void)pgShowInspector:(CDVInvokedUrlCommand *)command
 {
     [self runCommand:^(NSDictionary* argsDict) {
-        [SFSmartStoreInspectorViewController present:self.viewController];
+        BOOL isGlobal = [self isGlobal:argsDict];
+        SFSmartStoreInspectorViewController* inspector = isGlobal ? self.globalInspector : self.inspector;
```
Comment (wmathurin) [inline libs/SalesforceHybridSDK/SalesforceHybridSDK/Classes/Plugins/SFSmartStore/SFSmartStorePlugin.m]: This is the change this pull request is about

File: libs/SalesforceSDKCore/SalesforceSDKCore/Classes/SmartStore/SFSmartStoreInspectorViewController.m [hunk 3]
```diff
@@ -73,43 +73,41 @@ @interface SFSmartStoreInspectorViewController () <UINavigationBarDelegate>
 
 @implementation SFSmartStoreInspectorViewController
 
-#pragma mark - Singleton
+#pragma mark - Constructor
 
-+ (SFSmartStoreInspectorViewController *) sharedInstance
+- (instancetype) initWithStore:(SFSmartStore*)store
 {
-    static SFSmartStoreInspectorViewController *sharedInstance = nil;
```
Comment (wmathurin) [inline libs/SalesforceSDKCore/SalesforceSDKCore/Classes/SmartStore/SFSmartStoreInspectorViewController.m]: Not keeping a singleton anymore, but instead exposing a constructor that takes a store.

**Comment by bhariharan** [comment]
:+1:

**Comment by khawkins** [comment]
Looks good.

**Pull request merged**